core-ui/src/components/ApiRules/useGetGatewayDomain/useGetGatewayDomain.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useGet } from 'react-shared';
+
+const apiGatewayUri =
+  '/apis/networking.istio.io/v1beta1/namespaces/kyma-system/gateways/kyma-gateway';
+
+export function useGetGatewayDomain() {
+  const [domain, setDomain] = React.useState('');
+  const [loading, setLoading] = React.useState(true);
+  const [error, setError] = React.useState(null);
+
+  const { error: getError, loading: getLoading, data } = useGet(
+    apiGatewayUri,
```

Comment:
it's not an option for a namespace admin. not everyone has an access to the `kyma-system` namespace